Realm/Realm/Schema/PropertyType.cs

```diff
@@ -75,6 +75,11 @@ public enum PropertyType : ushort
         /// </summary>
         LinkingObjects = 8,
 
+        /// <summary>
+        /// A property that can contain RealmValue supported types.
+        /// </summary>
+        Mixed = 9,
```

Comment:
Since this is public, it should be `RealmValue`, not `Mixed`. We should never let Core terminology bleed into the public API.